### PR TITLE
Link items in docs and other improvements

### LINKED ISSUE #79: Hyperlink crate items in docs
Rust docs makes it easy to hyperlink doc items [by name](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html). This makes it easier to navigate the docs because the reader can quickly jump to the definition of an item being referred.

That being said hyperlinking everything everywhere can make the docs look noisy and hinder readability. This is somewhat subjective but some general rules of thumb -
* Only need to hyperlink items in this crate
* If an items is mentioned multiple times in a paragraph or couple of paragraphs, hyperlinking the first time it's mentioned is good enough
* If a many items from a module are enumerated (like in bulleted list) hyperlinking the module is good enough

_from discussion in #73_

### PR DESCRIPTION
**Pull request by twitu**
Closes #79 

Linking relevant items in docs and using simple present tense for docs where appropriate

### CODE REVIEW

File: src/utils.rs [hunk 1]
```diff
@@ -15,9 +15,9 @@ pub fn prefix_key(prefix: &dyn AsRef<[u8]>, key: &dyn AsRef<[u8]>) -> Vec<u8> {
     [prefix.as_ref(), key.as_ref()].concat()
 }
 
-/// Calculates the storage fee of an action, given an initial storage amount,
-/// and refunds the predecessor a portion of the attached deposit if necessary.
-/// Returns refund Promise if refund was applied.
```
Comment (nearken) [inline src/utils.rs:20]: I think plural tense is correct in this case

Comment (twitu) [reply src/utils.rs]: I took a look at a few other doc examples in other libraries including rust docs and simple present tense in plural is the most common style.

I've reverted this change.

**Comment by encody** [comment]
This ready for merge?

**Comment by twitu** [comment]
Yes no other changes needed.

**Pull request merged**